### PR TITLE
method_should_succeed moved to utility/utils

### PR DESCRIPTION
**Pull request by tintumathew10**
Signed-off-by: tintumathew10 <tmathew@redhat.com>

Moved method_should_succeed() to utility/utils.

Logs:
[Robust_rebalancing_-_osd_replacement_0.log](https://github.com/red-hat-storage/cephci/files/7819920/Robust_rebalancing_-_osd_replacement_0.log)
[startup.log](https://github.com/red-hat-storage/cephci/files/7819922/startup.log)

### CODE REVIEW

File: utility/utils.py [hunk 1]
```diff
@@ -1222,3 +1222,18 @@ def install_start_kafka(rgw_node, cloud_type):
...
+def method_should_succeed(function, *args, **kwargs):
+    """
+    Checks return status of command execution
+    and raise assertion on failure
+    Args:
+        function: name of the function
+        args: arg list
+        kwargs: arg dict
+    """
+    if function(*args, **kwargs):
```
Comment (psathyan) [inline utility/utils.py:1236]: Can we do
```
rc = function(*args, **kwargs)

Log.debug(f"The {function} return status is {rc}")
if not rc: raise AssertionError
```

File: utility/utils.py [hunk 2]
```diff
@@ -1222,3 +1222,18 @@ def install_start_kafka(rgw_node, cloud_type):
 
     # wait for kafka service to start
     time.sleep(30)
+
+
+def method_should_succeed(function, *args, **kwargs):
+    """
+    Checks return status of command execution
```
[the remark this answers is not in the document]
Comment (tintumathew10) [reply utility/utils.py]: added examples.

File: utility/utils.py (hunk 1 above)
Comment (tintumathew10) [reply utility/utils.py]: modified.

**Review by pdhiran — APPROVED** [review]
looks good

**Review by psathyan — APPROVED** [review]

**Pull request merged**